driver/pace/driver/grid.py

```diff
@@ -196,6 +199,110 @@ def get_grid(
         return damping_coefficients, driver_grid_data, grid_data
 
 
+@GridInitializerSelector.register("external")
+@dataclasses.dataclass
```

Comment:
Why `dataclass`? It's not necessarily wrong since it's a config, but it's not used that way, but more like a Functional Object.

Reply:
I had copied the other GridInitializer classes, should they also not be that way?

Reply:
Good point. Some of those looks much more like `dataclasses`.
Not a big deal anyway, as with everything in Python those "class type" are just syntactic sugar and not very constraining. Let's keep it `dataclasses` for uniformity.